<?xml version="1.0" ?>
<sdf version="1.4">
  <world name="default">
    <!-- A ground plane -->
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <!-- A global light source -->
    <include>
      <uri>model://sun</uri>
    </include>

    <include>
      <uri>model://atlas</uri>
    </include>

    <plugin filename="libHydra.so" name="hydra">
      <device>/dev/hidraw5</device>
      <pivot>0.04 0 0</pivot>
      <grab>0.12 0 0</grab>
    </plugin>

<!--
  <model name="simple_arm">
    <pose>0 0 0.05 0 0 0</pose>
    <link name="link1">
      <inertial>
        <mass>0.1</mass>
        <inertia>
          <ixx>0.001</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.001</iyy>
          <iyz>0.0</iyz>
          <izz>0.001</izz>
        </inertia>
      </inertial>

      <pose>0 0.5 0 0 0 0</pose>
      <collision name="collision">
        <pose>0.5 0 0 0 0 0</pose>
        <geometry>
          <box>
            <size>1 .1 .1</size>
          </box>
        </geometry>
      </collision>

      <visual name="visual">
        <pose>0.5 0 0 0 0 0</pose>
        <geometry>
          <box>
            <size>1 .1 .1</size>
          </box>
        </geometry>
      </visual>
    </link>

    <link name="link2">
      <pose>1.0 0.5 0 0 0 0</pose>
      <inertial>
        <mass>0.1</mass>
        <inertia>
          <ixx>0.001</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.001</iyy>
          <iyz>0.0</iyz>
          <izz>0.001</izz>
        </inertia>
      </inertial>

      <collision name="collision">

        <pose>.5 0 0 0 0 0</pose>
        <geometry>
          <box>
            <size>1 .1 .1</size>
          </box>
        </geometry>
      </collision>

      <visual name="visual">
        <pose>.5 0 0 0 0 0</pose>
        <geometry>
          <box>
            <size>1 .1 .1</size>
          </box>
        </geometry>
      </visual>
    </link>

    <joint name="joint1" type="revolute">
      <parent>world</parent>
      <child>link1</child>
      <axis>
        <xyz>0 0 1</xyz>
      </axis>
    </joint>

    <joint name="joint2" type="revolute">
      <parent>link1</parent>
      <child>link2</child>
      <axis>
        <xyz>0 0 1</xyz>
      </axis>
    </joint>

    <plugin name='simple_ik' filename='libSimpleIKPlugin.so'>
      <base_link>link1</base_link>
      <joint>joint1</joint>
      <joint>joint2</joint>
      <ef_pos>1 0 0</ef_pos>
    </plugin>
  </model>
  -->

  <model name="right_arm_goal">
    <static>true</static>
    <pose>0.0 -0.3 0.6 0 0 0</pose>
    <link name="link">
      <visual name="visual">
        <geometry>
          <box>
            <size>.1 .1 .3</size>
          </box>
        </geometry>
      </visual>
    </link>
  </model>

  <model name="left_arm_goal">
    <static>true</static>
    <pose>0.0 0.3 0.6 0 0 0</pose>
    <link name="link">
      <visual name="visual">
        <geometry>
          <box>
            <size>.1 .1 .3</size>
          </box>
        </geometry>
      </visual>
    </link>
  </model>

</world>
</sdf>
